src/inputs/Table/Table.js

```diff
@@ -168,21 +192,18 @@ export const Table = (props) => {
           buildErrorsCountLabel={buildErrorsPanelTitle}
         />
       )}
-      <div data-cy="grid" id="grid-container" style={dimensions} className={agTheme}>
+      <div data-cy="grid" id="grid-container" className={agTheme}>
         {isReady && (
-          <AgGridReact
-            ref={gridRef}
-            undoRedoCellEditing={true}
-            rowDragManaged={true}
-            suppressDragLeaveHidesColumns={true}
-            allowDragFromColumnsToolPanel={true}
-            columnDefs={formattedColumns}
-            defaultColDef={defaultColDef}
-            columnTypes={columnTypes}
-            rowData={rows}
-            context={context}
-            stopEditingWhenCellsLoseFocus={true}
-          />
+          <>
+            <TableToolbar isFullscreen={isFullscreen} toggleFullscreen={toggleFullscreen} />
```

Comment:
I'm not sure it's a good idea to pass `isFullScreen` and `toggleFullScreen` in props because `TableToolbar` doesn't need them and all it does it passing props to its child (so props drilling). Also, the toolbar is meant to have lots of children and it will be difficult to handle all children props via the toolbar. @ardvi  Maybe it's better to pass it by context or by Subject, like in `twoActionsDialog` ?

Reply:
I think for this simple case, props are good way.